scripts/build/package.js

```diff
@@ -49,18 +49,20 @@ const licenseToObject = (license) => {
   }
   return license;
 };
-const iconToObject = (icon) => {
+function iconToObject(icon) {
+  const escapedIconTitle = escape(icon.title);
   return util.format(
     iconObjectTemplate,
-    escape(icon.title),
+    escapedIconTitle,
     escape(icon.slug),
-    escape(icon.svg),
+    titleToHtmlFriendly(escapedIconTitle),
```

Comment:
I think this should be

```suggestion
    escape(titleToHtmlFriendly(icon.title)),
```

because `escape(x)` ensures any single quotes (`'`) in the string being embedded are escaped properly when the string is being embedded into a JavaScript template in the string [^1]. So, the escaping should happen after the title is transformed to a HTML friendly string to 1) avoid accidentally encoding what `escape` added and 2) ensure any single quote added by the encoding is escaped. (even if neither of these _should_ happen, and so it doesn't really matter, this suggestion is the _correct_ implementation I think so it's better to change for clarity).

[^1]: Example, `util.format("let x = '%s'", escape("foo'bar"))` results in `let x = 'foo\'bar'`.